main.py

```diff
@@ -1 +1,35 @@
-rint("hello")
+import mysql.connector
+from mysql.connector import Error
+from fastapi import FastAPI
+from pydantic import BaseModel
+import uvicorn
+
+app = FastAPI()
+
+db_config = {
+    "host": "35.239.158.202",
+    "user": "root",
+    "password": "7$x[;*X-0qpT[J2n",
+    "database": "test",
+}
+
+connection = mysql.connector.connect(**db_config)
+
+
+class StoreItem(BaseModel):
+    naziv: str
+    cena: int
+    kolicina: int
+
+
+@app.get("/store")
+def view_store():
+    cursor = connection.cursor()
+    cursor.execute("SELECT * FROM test.store")
```

Comment:
This string should be stored somewhere else. You can use some of this approaches:
1. ENUM class with constants where you will store all of the SQL queries
2. In some JSON file